src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -290,6 +291,53 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
     }
 
     /*********************************************************************************************************
+    * @description Verifies HouseHold RecordType is not the default RecordType for the current User.
+    */
+    @TestVisible
+    private void verifyAccountDefaultRecordType() {
+        try {
+           
+            Id npspHouseholdRecordType = UTIL_CustomSettingsFacade.getOrgContactsSettings().npe01__HH_Account_RecordTypeID__c;
+            Id npspOneToOneRecordType = UTIL_CustomSettingsFacade.getOrgContactsSettings().npe01__One_to_One_RecordTypeID__c;
+            Map<Id,Schema.RecordTypeInfo> accountRecordTypeInfosById = Account.SObjectType.getDescribe().getRecordTypeInfosById();
+
+            //Check if the Household and the One To One RecordRecord Type in the Settings is valid. 
+            //This should never happen, unless the User manually updated the Account Model Record Type in the Custom Settings.
+            if(
+                (npspHouseholdRecordType != null && accountRecordTypeInfosById.keySet().contains(npspHouseholdRecordType) == false) ||
+                (npspOneToOneRecordType != null && accountRecordTypeInfosById.keySet().contains(npspOneToOneRecordType) == false)
+            ) {
+                createDR(Label.healthLabelAccountDefaultRT, statusError, Label.healthDetailsAccountDefaultRTInvalid, Label.healthSolutionAccountDefaultRTInvalid);
+                return;
+            }
+
+            //Check the Household and One To One Record Type in the Settings and create a Failed Test if it is the Default one on the Account.
+            for(Schema.RecordTypeInfo accountRecordTypeInfo : accountRecordTypeInfosById.values()) {
+                if(
+                    accountRecordTypeInfo.isDefaultRecordTypeMapping() &&
+                    (
+                        accountRecordTypeInfo.getRecordTypeId() == npspHouseholdRecordType ||
+                        accountRecordTypeInfo.getRecordTypeId() == npspOneToOneRecordType
+                    )
+                ) {
+                    createDR(
+                        Label.healthLabelAccountDefaultRT, 
+                        statusError, 
+                        Label.healthDetailsAccountDefaultRTIssue, 
```

Comment:
This label has a {0} merge field, but that's not being replace here. The result is an error message such as the following:
![image](https://user-images.githubusercontent.com/801017/31289899-85d8642a-aa98-11e7-988c-c80100b11eac.png)

Reply:
Also, I don't think message explains what the actual error is. Maybe something like - "The default Household Record Type has been set to "Organization", which has been set as the default Record Type for the current user". I'm not sure this is all that clear either, but that's where @smarovich can help.